intro/scipy/index.rst

```diff
@@ -135,22 +135,64 @@ File input/output: :mod:`scipy.io`
 Special functions: :mod:`scipy.special`
 ---------------------------------------
 
-Special functions are transcendental functions. The docstring of the
-:mod:`scipy.special` module is well-written, so we won't list all
-functions here. Frequently used ones are:
+"Special" functions are functions commonly used in science and mathematics that
+are not considered to be "elementary" functions, including
```

Comment:
To avoid ambiguity over whether the list is of "special" functions or "elementary" functions:
```suggestion
"Special" functions are functions commonly used in science and mathematics that
are not considered to be "elementary" functions. Examples include
```